.travis.yml

```diff
@@ -23,12 +23,10 @@ deploy:
     overwrite: true
     on:
       tags: true
-  # TODO: Use 'travis encrypt --com -r adafruit/<repo slug>' to generate
-  #       the encrypted password for adafruit-travis. Paste result below.
   - provider: pypi
     user: adafruit-travis
     password:
-      secure: #-- PASTE ENCRYPTED PASSWORD HERE --#
+      secure: cBnTT/A6ePSz/eNidwxs9S6GamtLqc6z+a1GyV+Z0sNBtIyHmXIClvJKeGkJEQDtO6kHftNy+UhW6bQPrmRK8d3vcysHriLuFBWhViteBHNHSpacTXmbqbE3leavV0mHu3lX4moP2a3kYgk/kMNF6p9YNNyJPRshFpm930Wiq9Nn1zPh3ethjTY/EuFJt5KxxSzmCUYRqLSsGOV777UaPCIC4pft8smPIfTNcAmvFO1mlPFEikCbWg9fGq68PfJ5mEyvlgJrDS9ptEAIBNwu44+wMiVKVxit9PiifjY1FerAVP1s+Ka2IdkLB7JQZmBQMXJ51uLabRPfshwOhi28Lkc+0zJb3UzmTh6pXDUmfutlRAdVdY1HxfDpqzMvjt2FOGBMRm7fd0YUcIUMGX8SDbXoOL6fNeWAW3+ajiO2BTa+4cHrrn7q0UmnhRna3dos8jA9oGir2bkR97IYVjaJD9h3/BejMSiHXLRXSX7qTjh47JFWI9aW8eRGqhfXPnjsp1RZIizINJS3QIjysTZZCzSbMH+6fADqX820kU85w5gAol4zwveTAjoN3vGphyHgjFRcXe5lpqgUJXMveE3ywgwNvWheAQiniw+Tkl2mQ/zo9UwS6jwMJFyug+9FPrtxsn0pLZRPGEtnp/LbZfQuS1DZ6ZhL7tHjAHjm1rE0N+k=
```

Comment:
@rhooper @kattni, sorry I just noticed this. let's get this using the actions workflow instead of setting up travis only to have Dylan undo it.

Dylan already added the secrets to the repo, so this should just mean:

- Remove the `build*` line from the `.gitignore` so we can...
- Add the actions workflow files from `TestRepo`: https://github.com/adafruit/Adafruit_CircuitPython_TestRepo/tree/master/.github/workflows
- Update the build badge to look at Actions instead of Travis:
https://github.com/adafruit/Adafruit_CircuitPython_TestRepo/blame/master/README.rst#L12
- Remove `.travis.yml`

That should do it; if not let me know and I'll help

Reply:
@siddacious can you send a PR to this PR with the changes necessary?  I've run out of time today.

Reply:
Another important review note:

Awesome job! Looking forward to buf-ing _all_ the pixels

Reply:
@rhooper I will if I have time, otherwise Dylan can do it tomorrow

Reply:
i should have time tomorrow too.

Reply:
I think this is done now?  @siddacious can you check?

Reply:
@rhooper yup! I was just finishing up some morning stuff; looking now